.gitignore

```diff
@@ -74,3 +74,4 @@ terraform.tfstate*
 .idea
 mattermost-load-test.iml
 
+.terraform.tfstate.lock.info
```

Comment:
Ideally this shouldn't be there in your repo if everything went well. It only happens on a botched up deployment. But I guess it's good to keep it.